### ISSUE TITLE
neovide remote wrong

### ISSUE THREAD

**Issue by ian-ie** [LunarVim/LunarVim#3304]
### Problem description

1. Execute `lvim --listen 0.0.0.0:6666 --headless   ` on my arch
2. Execute `.\neovide.exe --remote-tcp 192.168.123.171:6666` on my win
3. 
![image](https://user-images.githubusercontent.com/55068959/196849728-99127684-0e7f-4708-b8e0-9bc8898e4065.png)

### LunarVim version

master-211bf4d 

### Neovim version (>= 0.8.0)

NVIM v0.8.0

### Operating system/version

Neovide 0.10.0

### support info

nil

**Comment by kylo252** [comment]
does this work with vanilla neovim?

just making sure it's not your firewall

**Comment by kobzar** [comment]
What about back way from arch to Neovide?

**Comment by ian-ie** [comment]
> What about back way from arch to Neovide?



> does this work with vanilla neovim?
> 
> just making sure it's not your firewall

Works fine. 
![image](https://user-images.githubusercontent.com/55068959/196882570-c404dedc-6ec4-47ca-906b-1b02749c0daf.png)

**Comment by ian-ie** [comment]
> What about back way from arch to Neovide?

When I use localhost in Arch, it does not change

**Comment by umop3plsdn** [comment]
if you do it with your regular nvim config instead of lunarvim does it work?

**Comment by ian-ie** [comment]
like this,It seems the same problem
![image](https://user-images.githubusercontent.com/55068959/196940832-753c9f3c-97b1-494d-b735-03d059e11f8a.png)

**Comment by kylo252** [comment]
@ian-ie, do you have a vanilla neovim config that you need to keep?

if not, just clone the this repo into `~/.config/nvim` and neovide will work as usual

**Comment by sunhs** [comment]
> @ian-ie, do you have a vanilla neovim config that you need to keep?
> 
> if not, just clone the this repo into `~/.config/nvim` and neovide will work as usual

@kylo252 Hi, I tried this but it didn't work. I cloned the master branch of LunarVim (commit 2d37303) in to `~/.config/nvim`, and using the vanilla nvim binary v0.9.0 (built from source). I started the server with `nvim --listen 0.0.0.0:6666 --headless` and connect to it with `nvim --server localhost:6666 --remote-ui`. The result was the same:
![image](https://user-images.githubusercontent.com/1916614/231048930-d380428a-b231-4bd7-ae9f-3c4ea7754eea.png)
